.circleci/config.yml

```
@@ -98,8 +109,11 @@ jobs:
     <<: *test-template
     docker:
       - image: circleci/python:3.7
-  linux-python-36:
+  linux-python-36-oldest:
     <<: *test-template
+    environment:
+      DEPENDENCIES: "OLDEST"
+      SIGNAC_VERSION: "signac"
```

I'd like to delete the pinning for `signac` from the "oldest" requirements and let that be handled by the `${SIGNAC_VERSION}` variable.
```suggestion
      SIGNAC_VERSION: "signac==1.0.0"
```